server/handlers/walletHandler/schemas.js

```diff
@@ -49,6 +49,14 @@ const walletPostSchema = Joi.object({
   about: Joi.string(),
 });
 
+const walletPatchSchema = Joi.object({
+  id: Joi.string().uuid().required(),
```

Comment:
id should not be here. The id should be gotten from the url /wallet/:id

Reply:
I removed it. Maybe I should better take it from the request (from the token) like in walletPost? It doesn't have id in the URL and I would expect Patch request to look similar to Post